parlai/core/torch_agent.py

```diff
@@ -701,7 +703,7 @@ def __init__(self, opt, shared=None):
         self.id = type(self).__name__.replace("Agent", "")
 
         # now set up any fields that all instances may need
-        self.EMPTY = torch.LongTensor([])
+        self.EMPTY = torch.zeros(0, dtype=torch.long)
```

Comment:
Interesting - is this a subtly different empty tensor than before?

Reply:
The difference is the type checker doesn't complain :D